app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsRobot.kt

```diff
@@ -177,7 +177,7 @@ class SettingsRobot {
         }
 
         fun openLanguageSubMenu(interact: SettingsSubMenuLanguageRobot.() -> Unit): SettingsSubMenuLanguageRobot.Transition {
-            scrollToElementByText("Language")
+            scrollToElementByText("Set as default browser")
```

Comment:
Is there any resource ID we can use here?

Reply:
Not applicable anymore.